src/content/docs/sdk/get-started/create-a-dropin.mdx

```diff
@@ -60,180 +91,235 @@ After generating the `.elsie.js` config, open it and take a look. Below is an an
 
-Let's take it for a spin! Run the following command to launch your project's development environment in its default state:
+Start the local dev server and preview your components.
+
+Run the following command to launch your project's development environment:
 
 ```bash
-yarn dev
 npm run dev
-pnpm run dev
 ```
 
-Congrats! You just launched your first composable frontend! Actually, no. What you're seeing is our frontend development environment. It's a preconfigured HTML page (`examples > html-host > index.html`) that loads your frontend components for testing during development.
+Congrats! You just launched your first drop-in component! Actually, what you're seeing is our frontend development environment. It's a preconfigured HTML page (`examples > html-host > index.html`) that loads your drop-in components for testing during development.
 
 <Diagram caption="Frontend development environment">
   ![Frontend development environment](../images/frontend.png)
 </Diagram>
 
-Now we're ready to start building a composable frontend. Stop the server with `ctrl + c` and let's get started.
-
+**Tip:**  
+Stop the server with `Ctrl + C`.
 </Task>
 
 <Task>
 ### Generate a new UI Component
 
+Scaffold a new UI component using the Elsie CLI.
+
 Start by executing the following command, replacing `<MyUiComponent>` with the name of the component you want to add. For a login form, you might choose, `npx elsie generate component --pathname LoginForm`.
 
 ```bash
 npx elsie generate component --pathname <MyUiComponent>
 ```
 
-Full details about UI component development will be provided soon. For now, let's take a quick look at the files that are generated for you:
+These files are preconfigured to work together. For example, the `LoginForm` component is automatically imported into `src/components/index.ts` so you can reference it throughout your project.
+
+If you run `npm run dev` again, you'll see your new component in the Storybook UI, configured with an example and best practices.
 
+**Example Output:**
 ```console
-~/composable-login [main] » npx elsie generate component --pathname LoginForm
 🆕 src/components/LoginForm/LoginForm.css created
 🆕 src/components/LoginForm/LoginForm.stories.tsx created
 🆕 src/components/LoginForm/LoginForm.test.tsx created
 🆕 src/components/LoginForm/LoginForm.tsx created
 🆕 src/components/LoginForm/index.ts created
 🆕 src/components/index.ts created
-~/composable-login [main] »
 ```
-
-These files were not only generated with the appropriate names, but they are completely preconfigured to work together as a unit. For example, the `LoginForm` component was automatically imported into `src/components/index.ts` to let you start referencing the component throughout your project.
-
-And if you run `npm run dev` again, you'll see your new component in the Storybook UI, configured with an example and best practices to help you get started with Storybook.
-
 </Task>
 
 <Task>
 ### Generate a new API Function
```

Comment:
Same as above: This topic needs more details about what a UI Component is in the context of drop-ins and what it's purpose is withing the drop-in.